homeassistant/components/scrape/sensor.py

```diff
@@ -39,38 +43,35 @@
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
+from .const import CONF_INDEX, CONF_SELECT, DEFAULT_NAME
 from .coordinator import ScrapeCoordinator
 
 _LOGGER = logging.getLogger(__name__)
 
 SCAN_INTERVAL = timedelta(minutes=10)
 
-CONF_ATTR = "attribute"
-CONF_SELECT = "select"
-CONF_INDEX = "index"
-
-DEFAULT_NAME = "Web scrape"
-DEFAULT_VERIFY_SSL = True
-
+# Old Platform Schema - falls under ADR-0007
 PLATFORM_SCHEMA = PARENT_PLATFORM_SCHEMA.extend(
     {
+        # Linked to the Scrape Resource
```

Comment:
Due to ADR0007 we are not allowed to change the schema.
I have only re-ordered the items to make it clearer what is linked to "loading the page" and what is linked to "parsing the page".